### PR TITLE
Fixed Maven test jars warnings

### PR DESCRIPTION
**Pull request by merlimat**
### Motivation

Removed unnecessary maven plugin configs that were generating build warnings.

### CODE REVIEW

File: pulsar-checksum/src/main/assembly/assembly.xml [hunk 1]
```diff
@@ -31,7 +31,6 @@
       <excludes>
         <exclude>org.slf4j:slf4j-api</exclude>
         <exclude>io.netty</exclude>
-        <exclude>com.scurrilous</exclude>
```
Comment (rdhabalia) [inline pulsar-checksum/src/main/assembly/assembly.xml]: is there any reason including this package?

Comment (merlimat) [inline pulsar-checksum/src/main/assembly/assembly.xml]: There is a `[WARNING]` message when compiling: 

```
[WARNING] The following patterns were never triggered in this artifact exclusion filter:
o  'com.scurrilous'
```

Comment (merlimat) [inline pulsar-checksum/src/main/assembly/assembly.xml]: I think that package is not present, hence the warning

**Comment by maskit** [comment]
retest this please

**Pull request merged**